health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/approveCommentPopUp.js

```
@@ -0,0 +1,96 @@
+        if (e.key === "Enter") {
+            handleSave();
+        }
+    };
+
+    return (
+        <>
+            <PopUp
+                style={{ width: "700px" }}
+                onClose={onClose}
+                heading={t(`HCM_AM_APPROVE_LABEL`)}
+                children={[
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)